src-backbone/app/js/views/builder/builderNavbarView.js

```diff
@@ -28,13 +33,12 @@ module.exports = Marionette.ItemView.extend({
         let self = this;
         this.model.destroy({
             success: function() {
-                console.info('Deleted procedure', self.model.get('id'));
+                console.info('Deleted Procedure', self.model.get('id'));
+                Helpers.navigateToDefaultLoggedIn();
```

Comment:
Are we doing notifications for only important things, not every action (i.e., this delete procedure action)